cypress/support/step-definitions/detail-steps.js

```diff
@@ -20,3 +21,7 @@ Then('icon not exists on preview', () => {
 Then('icon on preview is {string}', (iconName) => {
   icon().should('have.attr', 'data-element', iconName);
 });
+
+Then('icon name on preview is {string}', (iconName) => {
```

Comment:
we need to inform that this function is using only for iFrame rendered page

Reply:
fixed